packages/page/src/Infrastructure/Sulu/Admin/PageAdmin.php

```diff
@@ -125,25 +125,11 @@ public function configureViews(ViewCollection $viewCollection): void
                     'disabled_condition' => '(_permissions && !_permissions.edit)',
                 ]
             ),
-            'delete' => new DropdownToolbarAction(
+            'delete' => new ToolbarAction(
```

Comment:
@draconivis I guess this was unintentional? and the toolbaraction is also missing on snippet/articles